services/blockchain-indexer/shared/constants.js

```diff
@@ -100,6 +100,13 @@ const PATTERN_ANY_LOCAL_ID = '*'.repeat(LENGTH_LOCAL_ID);
 
 const MAX_COMMISSION = BigInt('10000');
 
+const KEY_VALUE_STORE_KEYS = {
```

Comment:
Referring to the innermost property becomes too long with the current naming. Eg: `KEY_VALUE_STORE_KEYS.PREFIX.TOTAL_LOCKED`.
```suggestion
const KV_STORE_KEYS = {
```

Reply:
Applied.